### DISCUSSION TITLE
Elementor doesn't support custom fields with Activecampaign

### DISCUSSION THREAD

**Discussion by leo26dandy** [elementor/elementor#16909] [Developers Community]
Hi Elementor!

I notice a limitation on your plugin when it comes to the form widget. 
What I do is to put custom field (Text Area) in the Elementor form and collect data like message.

So when I try to send that data, ActiveCampaign doesn't receive it because, as your docs say: "ActiveCampaign supports the following fields: Email, First Name, Last Name, Phone and Organization", meaning that it doesn't support custom fields.

It would really help my ActiveCampaign campaigns.

![AC- Elementor Issue](https://user-images.githubusercontent.com/25453370/141294792-032bcfbe-ae59-4c77-bade-6aba7a75b8fc.png)

**Comment by webboyon** [comment]
You should be A)  using the API for AC or B AC integrated forms 
I use CF forms pass the to to the API and this works fine 
if you need to  take it to another space use GTM and change anything you wish

**Comment by EmpoweredEO** [comment]
I user Memberium for this - works well
